### PR TITLE
Varios updates - For PR #1878

### PR DESCRIPTION
**Pull request by Furtif**
* Corrected typos @TheWizard1328 
* 0.67.2 update
* Rename GetGymDetails to GymGetInfo
* Fix show raids requires PRs :
Necrobot-Private/PokemonGo.RocketAPI#144
Necrobot-Private/PokemonGO-Assets#20 (or other)

### CODE REVIEW

**Comment by Lord-Haji** [comment]
Update this with RaidBuild

**Comment by Furtif** [comment]
is updated merge this to add chages to #1878

**Comment by Furtif** [comment]
sans_titre

File: PoGo.Necrobot.Window/Model/GymViewModel.cs [hunk 1]
```diff
@@ -30,34 +30,38 @@ public string GymIcon
             {
                 string fortIcon = "";
                 bool isRaid = false;
-                /*try
+                try
                 {
                     if (!string.IsNullOrEmpty(fort.RaidInfo.RaidPokemon.PokemonId.ToString()))
                         isRaid = true;
+                    else
```
Comment (Lord-Haji) [inline PoGo.Necrobot.Window/Model/GymViewModel.cs]: this else is useless

**Pull request merged**